x/data/msg_define_resolver_test.go

```diff
@@ -0,0 +1,51 @@
+package data
+
+import (
+	"testing"
+
+	"github.com/regen-network/gocuke"
+	"github.com/stretchr/testify/require"
+)
+
+type msgDefineResolverSuite struct {
+	t   gocuke.TestingT
+	msg *MsgDefineResolver
+	err error
+}
+
+func TestMsgDefineResolver(t *testing.T) {
+	gocuke.NewRunner(t, &msgDefineResolverSuite{}).Path("./features/msg_define_resolver.feature").Run()
+}
+
+func (s *msgDefineResolverSuite) Before(t gocuke.TestingT) {
+	s.t = t
+	s.msg = &MsgDefineResolver{}
+}
+
+func (s *msgDefineResolverSuite) AValidManager() {
+	s.msg.Manager = "cosmos1depk54cuajgkzea6zpgkq36tnjwdzv4afc3d27"
```

Comment:
this can go in the .feature file I think

Reply:
Moved to the feature file and within a message defined with json.